vendor.conf

```diff
@@ -87,7 +87,7 @@ golang.org/x/sys                                    9eafafc0a87e0fd0aeeba439a457
 golang.org/x/text                                   f21a4dfb5e38f5895301dc265a8def02365cc3d0 # v0.3.0
 golang.org/x/time                                   fbb02b2291d28baffd63558aa44b4b56f178d650
 google.golang.org/genproto                          02b4e95473316948020af0b7a4f0f22c73929b0e
-google.golang.org/grpc                              25c4f928eaa6d96443009bd842389fb4fa48664e # v1.20.1
+google.golang.org/grpc                              6eaf6f47437a6b4e2153a190160ef39a92c7eceb # v1.23.0
```

Comment:
Could you move the grpc and go-winio bumps to separate commits, if possible? Most of the other ones are probably highly coupled, so I guess this can stay in a single commit